crates/camera/src/camera.rs

```diff
@@ -209,19 +213,23 @@ impl DesiredPoW {
     }
 }
 
-fn setup(mut commands: Commands) {
+fn setup(mut commands: Commands, camera_query: Query<Entity, With<Camera3d>>) {
     commands.insert_resource(HorizontalMovement::default());
     commands.insert_resource(DesiredPoW {
-        distance: MAX_CAMERA_DISTANCE,
+        distance: DEFAULT_CAMERA_DISTANCE,
         off_nadir: Radian::ZERO,
         azimuth: Radian::ZERO,
     });
     commands.insert_resource(CameraFocus {
         point: Vec3::ZERO,
-        distance: MAX_CAMERA_DISTANCE,
+        distance: DEFAULT_CAMERA_DISTANCE,
     });
+    if let Ok(cam_entity) = camera_query.get_single() {
```

Comment:
This seems like an independent change which doesn't belong to this commit. I would remove it.